catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/endpoint/CometdEndpoint.java

```diff
@@ -45,7 +45,7 @@
 import ddf.catalog.filter.FilterBuilder;
 
 /**
- * The CometdEndpoint binds the SearchService and the CometdServlet together. 
+ * The CometdEndpoint binds the SearchService and the CometdServlet together.
```

Comment:
Use `{@link ClassName}` instead of just `ClassName` for all classes in this comment.